### DISCUSSION TITLE
How to retrieve the video path when stopping recording?

### DISCUSSION THREAD

**Discussion by ItsNiceCraft** [aatikturk/obsws-python#63] [Q&A] [answered]
I am trying to grab the path of the newly recoreded video file that OBS puts out when stopping a recording. I am currently using `cl.stop_record()` and this does not seem to be possible. [The StopRecord request returns the outputPath when called.](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#stoprecord) Would it be possible to implement this or find a workaround?

Thanks!

**Comment by onyx-and-iris** [comment, marked as answer]
Hi! Please check #7 for a working example.

It's also possible to get the recording filepath with the Events client. Check https://gist.github.com/onyx-and-iris/4b824ee93ac9dbbd701e069ee133b306.

I hope that is helpful.  I will mark this as duplicate.
